src/brasil/gov/agenda/tests/test_compromisso.robot

```diff
@@ -79,6 +76,20 @@ Test Edit Delete Compromisso AgendaDiaria
     Click Button  Excluir
     Wait Until Page Contains  Atualmente não existem compromissos agendados.
 
+Test Highlight Appointments
+    [Tags]  Expected Failure
+
+    Enable Autologin as  Site Administrator
+    Go to homepage
+
+    Create Agenda
+
+    Create  Compromisso  Compromisso do dia
+    Click Link  Agenda de Clarice Lispector para 28/10/2013
+    # I don't know why this is failing
```

Comment:
the test is failing because no days are highlighted.

![selenium-screenshot-1](https://user-images.githubusercontent.com/341393/41978727-cacf63dc-79f8-11e8-9514-eee43ef897da.png)

Reply:
@hvelarde could you build your static resources and try again please?